css-pseudo-4/Overview.bs

```diff
@@ -724,16 +718,68 @@ Default UA Styles</h3>
     /* Represent default spelling/grammar error styling in an adjustable way */
     :root::spelling-error { text-decoration-line: spelling-error; }
     :root::grammar-error  { text-decoration-line: grammar-error; }
-    /* Highlight targeted text */
-    :root::target-text    { color: MarkText; background: Mark; }
   </pre>
 
+  Some [=highlight pseudo-elements=]
+  should have <dfn>paired default highlight colors</dfn>,
+  consisting of a default 'color' and 'background-color'
```

Comment:
```suggestion
  should have <dfn>paired default highlight colors</dfn>--
  a default 'color' and 'background-color'
```

Reply:
done in 5084397ba8f811b011f8e6680489e7dff884298b